.travis.yml

```diff
@@ -30,6 +30,7 @@ before_install:
 
 install:
     - pip install -r ./.ci/python_requirements
+    - pip install -r ./.ci/auth_requirements
```

Comment:
Does Travis really need the authentication modules? Currently, the tests only test against naive credential list.